otx/api/utils/tiler.py

```diff
@@ -118,16 +122,25 @@ def predict_tile(
         """
         features = (None, None)
         offset_x, offset_y, tile_dict, tile_meta = self.preprocess_tile(image, coord)
-        raw_predictions = self.model.infer_sync(tile_dict)
-        output = self.model.postprocess(raw_predictions, tile_meta)
-        output = self.postprocess_tile(output, offset_x, offset_y, masks)
-        if return_features:
-            if "feature_vector" in raw_predictions or "saliency_map" in raw_predictions:
-                features = (
-                    raw_predictions["feature_vector"].reshape(-1),
-                    raw_predictions["saliency_map"][0],
-                )
-        return features, output
+        run_prediction = True
+        if not return_features:
+            # TODO[EUGENE]: should we set the threshold (0.45) as a parameter?
```

Comment:
@goodsong81 do you think we should set this confidence threshold as a configurable parameter? The higher the faster but risker.

Reply:
Yeah, I think it's good to be included in config params with default values.

Reply:
Hmm, or we just set the default good value for a while :)

Reply:
then should we leave it at 0.45?